views/component/courseDetails/template.njk

```diff
@@ -45,7 +45,7 @@
         text: "Level"
       },
       {
-        text: "AO to SEO",
+        text: grades,
```

Comment:
We'll want to put some form of default value in here if the course as no assigned grades:

e.g.
`text: grades if grades.length > 0 else "Not configured",`

Let me check whether 'Not configured' is the right terminology to use, get back to you with an answer on this tomorrow.